src/i18n/es/index.ts

```diff
@@ -0,0 +1,91 @@
+import type { BaseTranslation } from '../i18n-types';
+
+const es = {
+	allowConnections: 'Cambia la configuración de tu servidor para permitir conexiones desde',
+	allowMixedContent: 'Permitir contenido mixto',
+	areYouSureYouWantToDeleteAll: '¿Estás seguro de que quieres eliminar todos los {type:string}?',
+	areYouSureYouWantToLeave: '¿Estás seguro de que quieres salir?\nLa tarea en progreso se detendrá',
```

Comment:
Me suena raro "de que quieres", no?

```suggestion
	areYouSureYouWantToDeleteAll: '¿Estás seguro que quieres eliminar todos los {type:string}?',
	areYouSureYouWantToLeave: '¿Estás seguro que quieres salir?\nLa tarea en progreso se detendrá',
```

Reply:
No se por que me suena raro cuando lo leo la primera vez, pero me suena bien cuando lo leo una segunda vez 🤷 
Igual ok, que quede como esta 👍